pdb-alignment-pipeline/src/main/java/org/cbioportal/pdb_annotation/scripts/PdbScriptsPipelineInit.java

```diff
@@ -0,0 +1,18 @@
+package org.cbioportal.pdb_annotation.scripts;
+
+import org.springframework.boot.CommandLineRunner;
+import org.springframework.stereotype.Component;
+
+@Component
+public class PdbScriptsPipelineInit implements CommandLineRunner{
+	
+	
+	
+	@Override
+	public void run(String... args) throws Exception{
```

Comment:
@juexinwang: Does this method do actually something? Or its only purpose is to print "init" to stdout?